[2024-04-18 01:38:43,042] ERROR log_exception() L838  Exception on /consulta [DELETE] - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py L838 
Traceback (most recent call last):
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 883, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 902, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1174, in make_response
    raise TypeError(
TypeError: The view function for 'del_consulta' did not return a valid response. The function either returned None or ended without a return statement.
[2024-04-18 01:38:43,056] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 01:38:43] "[35m[1mDELETE /consulta?paciente=le HTTP/1.1[0m" 500 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:19:51,542] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:19:51] "OPTIONS /consulta?paciente=vvv HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:19:51,556] ERROR log_exception() L838  Exception on /consulta [DELETE] - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py L838 
Traceback (most recent call last):
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 883, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 902, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1174, in make_response
    raise TypeError(
TypeError: The view function for 'del_consulta' did not return a valid response. The function either returned None or ended without a return statement.
[2024-04-18 15:19:51,564] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:19:51] "[35m[1mDELETE /consulta?paciente=vvv HTTP/1.1[0m" 500 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:19:56,071] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:19:56] "OPTIONS /consulta?paciente=eee HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:19:56,086] ERROR log_exception() L838  Exception on /consulta [DELETE] - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py L838 
Traceback (most recent call last):
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 883, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 902, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1174, in make_response
    raise TypeError(
TypeError: The view function for 'del_consulta' did not return a valid response. The function either returned None or ended without a return statement.
[2024-04-18 15:19:56,094] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:19:56] "[35m[1mDELETE /consulta?paciente=eee HTTP/1.1[0m" 500 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:19:59,125] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:19:59] "OPTIONS /consulta?paciente=ff HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:19:59,143] ERROR log_exception() L838  Exception on /consulta [DELETE] - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py L838 
Traceback (most recent call last):
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 883, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 902, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1174, in make_response
    raise TypeError(
TypeError: The view function for 'del_consulta' did not return a valid response. The function either returned None or ended without a return statement.
[2024-04-18 15:19:59,148] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:19:59] "[35m[1mDELETE /consulta?paciente=ff HTTP/1.1[0m" 500 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:20:02,630] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:20:02] "OPTIONS /consulta?paciente=c HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:20:02,644] ERROR log_exception() L838  Exception on /consulta [DELETE] - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py L838 
Traceback (most recent call last):
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 883, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 902, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\flask\app.py", line 1174, in make_response
    raise TypeError(
TypeError: The view function for 'del_consulta' did not return a valid response. The function either returned None or ended without a return statement.
[2024-04-18 15:20:02,644] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:20:02] "[35m[1mDELETE /consulta?paciente=c HTTP/1.1[0m" 500 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
[2024-04-18 15:20:07,401] INFO _log() L96   127.0.0.1 - - [18/Apr/2024 15:20:07] "OPTIONS /consulta?paciente=lethi HTTP/1.1" 200 - - call_trace=C:\Users\lethi\OneDrive\Área de Trabalho\desenvolvimento-full-stack-basico\mvp\minha_api\env\Lib\site-packages\werkzeug\_internal.py L96  
